### DISCUSSION TITLE
Why should x64-windows-static-md not be the preferred triplet on Windows ?

### DISCUSSION THREAD

**Discussion by ankurvdev** [microsoft/vcpkg#20706] [Q&A]
Windows (Win32) currently has the following triplets
x64-windows 
x64-windows-static
x64-windows-static-md (community)

and then there's the following uwp triplet which maybe very well be considered separate platform but because of similiar build + consuming semantics I'm mentioning here.
x64-uwp (dyamic-lib, dynamic-crt)

The topic has been touched upon in another issue #15122  where the discussion originated but that issue was more targeted towards gathering enough consensus to at-least enable it as a tested-port. in the CI/CD Pipelines.

There's been some interest in carrying the discussion forward and documenting details around which triplet is best suited on windows

I think a whitepaper on this would go a long way to establish some knowledge base of the issues and debunking myths.

Based on my anecdotal evidence on ease of use for development, debugging, packaging 
I would strongly recommend the following
x64-windows-static-md : preferred triplet
x64-uwp (static-lib dynamic-crt)
And would like to know why that is contrary (if at all) to what vcpkg recommends.

In both these cases, DLL management and packaging is a huge - nightmare and more so in x64-uwp which its is next to impossible getting the right ingredients to produce an msix/appx with multiple dlls from their libs.

**Comment by BillyONeal** [comment]
I don't believe our position has changed from https://github.com/microsoft/vcpkg/issues/15122 to which you linked.

**Comment by ankurvdev** [comment]
@BillyONeal , This is more about documenting why the position is what it is.
Most of us don't understand the reasoning behind that or the caveats around the inherent risks to make an informed decisions as to which of them apply.

A whitepaper by the vcpkg team on why it is what it is would help us understand it a better
